lib/twitter-ads/client.rb

```diff
@@ -16,15 +17,17 @@ class Client
     # @param consumer_secret nil [String] The application consumer secret value.
     # @param access_token nil [String] The access token value.
     # @param access_token_secret nil [String] The access token secret value.
+    # @param opts [Hash] An optional Hash of extended options.
     #
     # @since 0.1.0
     #
     # @return [Client] The newly created client instance.
-    def initialize(consumer_key, consumer_secret, access_token, access_token_secret)
+    def initialize(consumer_key, consumer_secret, access_token, access_token_secret, opts = {})
```

Comment:
I'm going to call the options here `trace` for request tracing and `sandbox` for sandbox mode.